### ISSUE TITLE
Floating menu: Hide on drag/resize/rotate

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#11014]
## Feature Description

Request from design after playing with floating menus as they currently exist ahead of release 1.19. 

Hide the active floating menu of an element when that element is being dragged, resized, or rotated - menu to show back up once action is complete.

**Comment by felipebochehin87** [comment]
Tested against https://stories-qa-wordpress-amp.pantheonsite.io/wp-admin/, using PR #11057.

1. Created a new story
2. Added an image
3. Rotated image - floating menu was hidden
4. Resized image - floating menu was hidden
5. Dragged and dropped the image to move it - floating menu was hidden

Tested in Safari, Chrome and Firefox.

https://images.zenhubusercontent.com/235435637/0a542877-34ea-421e-bce1-beaea1c2c6df/11014.mp4
